src/Symfony/Component/Scheduler/Exception/AlreadyScheduledTaskException.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Scheduler\Exception;
+
+/**
+ * @author Guillaume Loulier <contact@guillaumeloulier.fr>
+ */
+final class AlreadyScheduledTaskException extends \RuntimeException
```

Comment:
maybe you should/could create a base class ex of the component
so that an external dev can catches all ex from this component easily